src/couch_mrview/src/couch_mrview_http.erl

```diff
@@ -473,12 +474,21 @@ parse_params(Props, Keys, Args) ->
 
 parse_params(Props, Keys, #mrargs{}=Args0, Options) ->
     IsDecoded = lists:member(decoded, Options),
-    % group_level set to undefined to detect if explicitly set by user
-    Args1 = Args0#mrargs{keys=Keys, group=undefined, group_level=undefined},
+    Args1 = case lists:member(keep_group_level, Options) of
+                true ->
```

Comment:
Style nit. Your case should be indented like such:

```erlang
Args1 = case lists:member(...) of
    true ->
        Args0;
    _ ->
        % group_level set to undefined ...
        Args0#mrargs{...}
end,
```

Reply:
Fixed in cf89a51